extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -103,5 +110,74 @@ describes.realWin(
+            /* Very, Very, Very long string value - will not pass string length test */
+            'productPrice': 19,
+            'productPriceCurrency': 'JPY',
+            'productImages': [
+              '/examples/visual-tests/amp-story/img/shopping/nest-mini-icon.png',
+              '/examples/visual-tests/amp-story/img/shopping/nest-mini-icon.png',
+            ],
+            'reviewsPage': 'https://store.google.com/jp/?hl=ja',
+            'brandLabel': 'Google',
+            'productDetails': 'Vinyl for the hit J-Pop song Plastic Love.',
+          },
+        ],
+      };
+
+      await createAmpStoryShoppingConfig(invalidConfig);
+    });
+
+    it('test config not a number', async () => {
+      expectAsyncConsoleError(userWarnStub, 1);
+      const invalidConfig = {
+        'items': [
+          {
+            'productId': 'city-pop',
+            'productTitle': 'Plastic Love',
+            'productPrice': 'two dozen watermelons',
+            /* Not an actual price */
+            'productPriceCurrency': 'JPY',
+            'productImages': [
+              '/examples/visual-tests/amp-story/img/shopping/nest-mini-icon.png',
+              '/examples/visual-tests/amp-story/img/shopping/nest-mini-icon.png',
+            ],
+            'reviewsPage': 'https://store.google.com/jp/?hl=ja',
+            'brandLabel': 'Google',
+            'productDetails': 'Vinyl for the hit J-Pop song Plastic Love.',
+          },
+        ],
+      };
+
+      await createAmpStoryShoppingConfig(invalidConfig);
+    });
+
+    it('test config invalid url array', async () => {
+      expectAsyncConsoleError(userWarnStub, 1);
+      const invalidConfig = {
+        'items': [
+          {
+            'productId': 'city-pop',
+            'productTitle': 'Plastic Love',
+            'productPrice': 19,
+            'productPriceCurrency': 'JPY',
+            'productImages': ['http://pizazz', 'http://zapp'],
+            'reviewsPage': 'http://zeppelinzero.com',
+            'brandLabel': 'Google',
+            'productDetails': 'Vinyl for the hit J-Pop song Plastic Love.',
+          },
+        ],
+      };
+
+      await createAmpStoryShoppingConfig(invalidConfig);
```

Comment:
Can you write assertions on the stub to make sure it's been called with the right error?